README.md

```diff
@@ -127,16 +127,22 @@ curl --location --request POST 'localhost:9933' \
     "id": 1
 }'
 ```
-#### Connecting to Polkadot Apps
+#### Interacting using Polkadot JS Apps
```

Comment:
I think that you should merge `main` into your branch so that this change does wont show up